main.tf

```
@@ -95,6 +103,14 @@ locals {
       valueFrom = var.webhook_ssm_parameter_name
     },
   ] : []
+  
+  # Azure requires basic auth for webhooks
+  container_definition_secrets_3 = [
```

I'd suggest more specific name:
```suggestion
  container_definition_secrets_azure_devops = [
```